docs/layers/base-layer.md

```diff
@@ -147,25 +129,26 @@ that are not Web Mercator projected longitudes/latitudes.
 
 ##### `projectionMode` (Number, optional)
 
-Specifies how layer positions and offsets should be interpreted.
+Specifies how layer positions and offsets should be geographically interpreted.
```

Comment:
We support a non-geographical mode, so maybe clearer to say 
Specifies how layer positions and offsets should be interpreted (as geospatial or mathematical coordinates).